src/Elmish/Hooks/Type.purs

```diff
@@ -70,18 +87,22 @@ type Hook = Cont ReactElement
 -- | ```
 mkHook :: forall msg state a. ComponentName -> ((a -> ReactElement) -> ComponentDef msg state) -> Hook a
 mkHook name mkDef =
-  cont \render -> wrapWithLocalState name mkDef render
+  Hook \render -> wrapWithLocalState name mkDef render
+
+-- | Unwraps a `Hook ReactElement`
```

Comment:
I think this might need a bit more comments, perhaps with an example, or at least referring to the `Hook` comments.